### ISSUE TITLE
Life stage facet

### ISSUE THREAD

**Issue by GillBrown** [hiscom/avh-testing#2]
Under the life stage facet there are some loaclity/place names. For example when you search _Archidendron_ you get features you would expect like flowers, fertile, fruit, etc. But you also get bellenden, tully, daintree, herbert, etc.
![image](https://cloud.githubusercontent.com/assets/17188858/17196742/2b0ab646-54aa-11e6-89e5-1669b49dd566.png)

**Comment by ryonen** [comment]
I've noted this issue for other taxa too. It appears to be a data delivery issue from individual participating herbaria, where locality information has accidentally been entered in the 'Phenology' field. See these records for Gill's Archidendron examples: http://avh-test.ala.org.au/occurrences/search?q=lsid%3Ahttp%3A%2F%2Fid.biodiversity.org.au%2Fnode%2Fapni%2F2900791&qc=data_hub_uid:dh9&fq=(life_stage%3A%22bloomfield%22%20OR%20life_stage%3A%22daintree%22%20OR%20life_stage%3A%22innisfail%22%20OR%20life_stage%3A%22tully%22%20OR%20life_stage%3A%22bellenden%22%20OR%20life_stage%3A%22hastings%22%20OR%20life_stage%3A%22herbert%22%20OR%20life_stage%3A%22ker%22%20OR%20life_stage%3A%22lamb%22%20OR%20life_stage%3A%22leafless%22%20OR%20life_stage%3A%22macleay%22%20OR%20life_stage%3A%22paluma%22%20OR%20life_stage%3A%22seaview%22)#tab_recordsView
Are these just 'Flag an issue' issues for the data provider?

**Comment by ben3000** [comment]
I think that is a good place to start. (And by that I meant contact the data provider.)

**Comment by nielsklazenga** [comment]
Please note that life stage is not phenology. Phenology is reproductiveCondition (or that's what we have mapped it to in HISPID). The life stage field should be deleted from AVH, as so far nobody has delivered it (we only put it in HISPID last year), so everything that is in there is wrong.

There are 1,797,743 records that have life_stage in AVH (http://avh-test.ala.org.au/occurrences/search?q=*%3A*&fq=life_stage:*). Have fun annotating.

**Comment by ryonen** [comment]
Niels - if you could enlighten me? What was 'life stage' supposed to cover? Plants juvenile/mature/dead [population survey-type criteria]? Also, where locality information appears in the 'life stage' facet (as for Archidendron egs above) why does it appear in the Phenology field for the actual specimen record? Do 'life stage' and reproductiveCondition map to the same place? 
I see in the AVH help for Download Fields, Phenology is just phenology and it's noted that only NSW and MEL deliver that data to AVH...

**Comment by nielsklazenga** [comment]
Yes, that sort of thing. The only thing I can imagine we (MEL) would use it for is seedlings (as we have a special collection of those from the seed bank thing).

Phenology was first delivered to ALA as Phenology, but was later was mapped to the Darwin Core reproductiveCondition. Because of the NULL value issue, and because we were going to review HISPID, there was no point in re-uploading the entire data set. Now that the HISPID review is over and the NULL value issue is fixed, we can do that now (and it is high time, as there are many other issues). We haven't been able to keep the documentation up-to-date and phenology is one thing we missed.

In issue #1, I suggested that reproductiveCondition might also not be a good fit for what we call 'phenology' (presence of certain reproductive (or non-reproductive) structures).
